src/handbook/customer/marketing/blog.md

```diff
@@ -21,7 +21,7 @@ Content creation and blogging are an effective way to communicate with our commu
 8. Content will be reviewed first.
 9. SEO will be reviewed second.
 10. When both reviews are done, the Author or SEO reviewer can merge. In the event of an external author, SEO/content review will merge.
-11. Once merged, SEO Reviewer will promote on FlowFuse Social channels.
+11. Once merged, SEO Reviewer will promote on FlowFuse Social channels. Check the [social media planning and posting section](/handbook/customer/marketing/social-media/#planning-and-posting) for guidance and considerations.
```

Comment:
This does work. So we create a PR, and start the relevant processes. If one process is slower than others, that's tough, but is something that should be fixed in the process -- not by delaying progress elsewhere.

Reply:
This comment was a follow up on [this one](https://github.com/FlowFuse/website/pull/1913#discussion_r1557069086), right?